lib/faulty/storage/redis.rb

```diff
@@ -392,7 +394,18 @@ def check_client_options!
       end
 
       def check_redis_options!
-        ropts = redis { |r| r.instance_variable_get(:@client).options }
+        ropts = redis do |r|
+          client = r.instance_variable_get(:@client)
+
+          if client.is_a?(::Redis::Cluster)
+            warn <<~MSG
+              Redis Cluster is not an appropriate storage backend for Faulty.
+              See https://github.com/ParentSquare/faulty/pull/51#issuecomment-1045280867
```

Comment:
Let's write some documentation on this subject, perhaps in the top-level class docs instead of linking to GitHub. "See API docs for Faulty::Storage::Redis".